src/CoreWCF.UnixDomainSocket/src/CoreWCF/Channels/UnixDomainSocketTransportBindingElement.cs

```diff
@@ -0,0 +1,110 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Authentication.ExtendedProtection;
+using CoreWCF.Configuration;
+
+namespace CoreWCF.Channels
+{
+    public class UnixDomainSocketTransportBindingElement : ConnectionOrientedTransportBindingElement
+    {
+        private int _listenBacklog;
+        private ExtendedProtectionPolicy _extendedProtectionPolicy;
+
+        public UnixDomainSocketTransportBindingElement() : base()
+        {
+            _listenBacklog = UnixDomainTransportDefaults.GetListenBacklog();
+            _extendedProtectionPolicy = ChannelBindingUtility.DefaultPolicy;
```

Comment:
Does ExtendedProtectionPolicy get plumbed through to WindowsStreamSecurityBindingElement? It's used for setting up NegotiateStream. I'm not sure if this class is where this should be configured or if it should be set directly on WindowsStreamSecurityBindingElement